tanjun/parsing.py

```diff
@@ -958,18 +958,22 @@ def set_parameters(
 
         return self
 
-    def bind_client(self, client: tanjun_abc.Client, /) -> None:
+    def bind_client(self: _ShlexParserT, client: tanjun_abc.Client, /) -> _ShlexParserT:
```

Comment:
Neither this nor the `bind_component` method were following the ABC, I assume this was an accident.